docs/add-ons/tutorial.md

```diff
@@ -126,13 +126,14 @@ Add "ports" to `config.json`. This will make TCP on port 8000 inside the contain
   "slug": "hello_world",
   "description": "My first real add-on!",
   "arch": ["armhf", "armv7", "aarch64", "amd64", "i386"],
-  "startup": "before",
+  "startup": "application",
   "boot": "auto",
+  "ingress": true,
+  "ingress_port": 8000,
```

Comment:
By default, we don't use the `ingress_port` and recommend to use the default port.

Reply:
Yes, but the example starts the python http server on port 8000